test/test_run_analysis.rb

```diff
@@ -476,6 +476,8 @@ def _verify_outputs(cli_output_log)
       next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and weekday fractions provided; the latter will be ignored.")
       next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and weekend fractions provided; the latter will be ignored.")
       next if _expected_warning_message(message, "Both 'lighting_garage' schedule file and monthly multipliers provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'cooling_setpoint' schedule file and cooling setpoint temperature provided; the latter will be ignored.")
+      next if _expected_warning_message(message, "Both 'heating_setpoint' schedule file and heating setpoint temperature provided; the latter will be ignored.")
```

Comment:
These warnings are now thrown for every datapoint because the HPXML has _both_ a constant heating/cooling setpoint temperature _and_ a detailed heating/cooling setpoint temperature schedules (as created by ResStockArgumenstPostHPXML)?

Reply:
ResStockArgumenstPostHPXML won't create detailed heating/cooling setpoint unless doing load flexibility.